hrm-service/service-tests/db-init/migrate.js

```diff
@@ -27,14 +27,16 @@ async function migrate() {
   let ret;
   let lastErr;
   console.log(timeoutPoint);
+  console.log('Umzug migration starting.');
```

Comment:
Just a nit fix. I slowed this down a bit and reduced the stream of text output during db startup.